src/client/pages/Profile.js

```diff
@@ -0,0 +1,149 @@
+import React, { useEffect, useState } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import Navbar from '../components/Navbar';
+import useFetch from '../hooks/useFetch';
+
+import Button from '../components/Button';
+
+import '../style/Profile.css';
+
+function Profile() {
+  const location = useLocation();
+  const history = useHistory();
+  const [username, setUsername] = useState();
+  const [email, setEmail] = useState();
+  const [about, setAbout] = useState();
+  const [name, setName] = useState();
+  const [loc, setLocation] = useState();
+  const [verified, setVerified] = useState();
+  const [DOB, setDOB] = useState();
+  const [ID, setID] = useState(2);
+
+  const endPoint = `/api/getProfile/${ID}`;
+  const [isLoading, data, error] = useFetch(endPoint);
+  const dataList = data || [];
```

Comment:
The naming here does not make sense. Is it a list or a single object?

Reply:
I have a number of fields in the user profile and in the DB  (columns of a single row of a table), each field is a constant here, dataList is either data from API or empty data. Taking this into account, how can I improve the implementation?